server/models/userProfile.js

```diff
@@ -0,0 +1,43 @@
+const mongoose = require('mongoose')
+
+const reqString = {
+  type: 'string',
+  required: true
+}
+
+const noReqString = {
+  type: 'string',
+  required: false
+}
+
+const date = {
+  type: Date,
+  default: null
+}
+
+const userProfile = new mongoose.Schema(
+  {
+    firstName: reqString,
+    lastName: reqString,
+    gender: reqString,
+    birthDay: date,
+    email: reqString,
+    phoneNumber: reqString,
+    address: reqString,
+    city: noReqString,
+    country: noReqString,
+    description: noReqString,
+    available: { type: 'boolean', required: true },
+    availableDays: [
+      {
+        availableDay: date,
+        from: noReqString,
+        to: noReqString,
+      }
+    ]
+  },
+  { timestamps: true }
+)
```

Comment:
These PR should have been made together with https://github.com/hatchways/team-pumpernickel/pull/52. If you add userId ref in profile, it would be much easier see the comments on that PR
    ref: "User",
    required: true,
  },_